src/main/java/com/targomo/client/api/request/config/RequestConfigurator.java

```diff
@@ -614,8 +614,12 @@ private static JSONObject getTravelMode(final TravelOptions travelOptions,
         }
 
         // snapping parameters
-        if (travelOptions.getIncludeSnapDistance() != null)
+        if (travelOptions.getIncludeSnapDistance() != null) {
             travelMode.put(INCLUDE_SNAP_DISTANCE, travelOptions.getIncludeSnapDistance());
+            if (travelOptions.getIncludeSnapDistance()) {
```

Comment:
```
if (travelOptions.getIncludeSnapDistance() && travelOptions.getWalkSpeed() != null) {
```
but probably works without the change

Reply:
changed to always include the walk speed for bike and car modes, because if are snapping is included the walk speed is considered even without includeSnapDistance (as inconvenience).